internal/handlers/jsonb1/where.go

```diff
@@ -142,6 +142,13 @@ func fieldExpr(field string, expr *types.Document, p *pg.Placeholder) (sql strin
 			// {field: {$gte: value}}
 			sql += "_jsonb->" + p.Next() + " >="
 			argSql, arg, err = scalar(value, p)
+		case "$size":
+			// { field: { $size: 2 } }
+			sql += "jsonb_array_length(_jsonb::jsonb->" + p.Next() + ") = "
```

Comment:
I _think_ `::jsonb` casting should not be required there – `_jsonb` column already has this type.

Reply:
I totally agree.